scripts/ci/install-minikube.sh

```diff
@@ -46,33 +45,52 @@ check_minikube() {
 }
 
 install() {
+	set -x
 	install_binary minikube $MINIKUBE_URL
 	install_binary kubectl $KUBECTL_URL
+	set +x
 }
 
 uninstall() {
+	set -x
 	uninstall_binary minikube
 	uninstall_binary kubectl
+	set +x
 }
 
 stop() {
+	set -x
 	check_minikube
-	sudo $(which minikube) delete || true
+	sudo minikube delete || true
 	sudo rm -rf ~/.minikube ~/.kube
+	set +x
 }
 
 start() {
+	set -x
 	check_minikube
-	sudo -E CHANGE_MINIKUBE_NONE_USER=true $(which minikube) \
-		--vm-driver=${MINIKUBE_VM_DRIVER} \
-		${MINIKUBE_FLAGS} start
-	sudo $(which minikube) status
+	export CHANGE_MINIKUBE_NONE_USER=true
+	if [ -z "$MINIKUBE_FLAGS" ]; then
+		MINIKUBE_FLAGS="
+			--vm-driver=none
+			--extra-config=kubelet.CgroupDriver=systemd
+			"
+	fi
+	sudo minikube start ${MINIKUBE_FLAGS}
+	sudo rm -rf ~/.minikube ~/.kube
+	sudo mv /root/.minikube /root/.kube ~
```

Comment:
why/how would root would have the right configuration ?
and how it will use it as you are starting the service minikube before ?

Reply:
correct - will be changed to 'cp'